trading_calendars/exchange_calendar_xhkg.py

```diff
@@ -92,6 +92,7 @@ class XHKGExchangeCalendar(TradingCalendar):
     Exchange calendar for the Hong Kong Stock Exchange (XHKG).
 
     Open Time: 9:31 AM, Asia/Hong_Kong
+    Lunch Break: 12:00 PM - 1:00 PM Asia/Hong_Kong
```

Comment:
At least until we change our start time labels as @rsheftel was suggesting, shall we make this consistent?
```suggestion
    Lunch Break: 12:01 PM - 1:00 PM Asia/Hong_Kong
```